pyproject.toml

```diff
@@ -183,7 +183,7 @@ rest_framework = ["rest_framework"]
 [tool.pytest.ini_options]
 DJANGO_SETTINGS_MODULE = "config.settings"
 testpaths = ["rdmo"]
-python_files = "test_*[!.txt].py"
+python_files = "test_*.py"
```

Comment:
@MyPyDavid @afuetterer for some reason I still don't understand, this change removed a substantial part of our tests from the CI. The tests run, but they are not included in just running `pytest`. This concerned import and export. I should have investigated when the coverage dropped after 2.0.

Reply:
ok, oops 🙈 , strange but good to know

Reply:
Which one was problematic?
test_*.py or test_*[!.txt].py?

What is the meaning of test_*[!.txt].py?